python/cugraph/jaccard/test_jaccard.py

```diff
@@ -145,3 +145,26 @@ def test_jaccard_edgevals(graph_file):
 
     print("Mismatches:  %d" % err)
     assert err == 0
+
+@pytest.mark.parametrize('graph_file', DATASETS)
+def test_jaccard_two_hop(graph_file):
+    M = read_mtx_file(graph_file)
+    M = M.tocsr()
+    Gnx = nx.DiGraph(M).to_undirected()
+    G = cugraph.Graph()
+    row_offsets = cudf.Series(M.indptr)
+    col_indices = cudf.Series(M.indices)
+    G.add_adj_list(row_offsets, col_indices, None)
```

Comment:
We need to test on edge lists too in 0.7.

Reply:
I will add a test.

Reply:
Done.